test/serviceprofiles/serviceprofiles_test.go

```diff
@@ -73,6 +73,7 @@ func TestServiceProfiles(t *testing.T) {
 
 	for _, tc := range testCases {
 		tc := tc // pin
+		t.Skipf("Skipping TestServiceProfiles/%s", tc.sourceName)
```

Comment:
rather than skipping the whole test, what about replacing some or all of the `t.Fatalf`'s below with `t.Skipf`? is there a particular case that's failing so we can only skip a single check?

Reply:
Based on [these](https://travis-ci.org/linkerd/linkerd2/jobs/523726427#L1050) integration logs, it looks like all the tests were failing for some reason. The failures are attributed to when service profiles are applied but do not show up immediately in the `linkerd routes` command. Since the command is pretty much used in each test, I thought disabling it this way might be better.

Reply:
Have you tried restarting the CI job to see if that helps? Looks like the log tests caught (and skipped) a bunch of timeout errors prior to the service profile tests.

Reply:
@adleong initially encountered this issue first, and then tried restarting but didn't seem to self-correct. Weird that the some tests were skipped, which tests were they @ihcsim?

Reply:
The `TestLogs()` test function skips, but outputs, unexpected errors. I was just surprised to see the number of timeout errors in your integration logs; possibly indicating an unhealthy cluster.

Looks like the integration test just [passed](https://gist.github.com/l5d-bot/3be5b6475d9bff94a448f79b8aba425a)?

Reply:
The integration tests that just passed are for this brunch which skips the service profile tests entirely. However, I do see `TestLogs()` skipping tests on this branch as well. I too suspect that the cluster is out of wack, pods are not being scheduled or something. it's a bummer that these tests depends on how healthy a cluster is. I'd need to spend some time thinking about a more reliable way to test this functionality 🤔

Reply:
Based on [that test log](https://travis-ci.org/linkerd/linkerd2/jobs/523726427#L1050), it looks like we could instead replace the `t.Errorf`'s on lines 243 (`assertRouteStat`) and 250 + 266 (`assertExpectedRoutes`) with `t.Skipf`'s. This way, we're still exercising the `linkerd profile` command (and that at least `linkerd routes` does not return an error). The intent here is to skip the narrowest set of testing possible.

Reply:
Ah, FWIW re: `TestLogs()`, we recently enabled rendering of all log failures via #2724, previously we were just hiding them, so things are not necessarily worse, it's just the logging issues are more visible. =)